<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PedalInterface</class>
 <widget class="QMainWindow" name="PedalInterface">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>544</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Pedal Interface</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Live Data</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Raw</string>
     </property>
     <widget class="QSpinBox" name="GasRawReading">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>8388607</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="BrakeRawReading">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>8388607</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ClutchRawReading">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Mapped</string>
     </property>
     <widget class="QSpinBox" name="BrakeMap">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-1023</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ClutchMap">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>-1023</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="GasMap">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-1023</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QGroupBox" name="ClutchGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Clutch</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSpinBox" name="ClutchHighSpinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ClutchLowSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
     <widget class="QSlider" name="ClutchLowSlider">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="ClutchHighSlider">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>30</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="BrakeGroupBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Brake</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSpinBox" name="BrakeHighSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
      <property name="value">
       <number>800000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="BrakeLowSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSlider" name="BrakeLowSlider">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="BrakeHighSlider">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>30</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
      <property name="value">
       <number>800000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GasGroupBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>101</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Gas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QSpinBox" name="GasHighSpinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSlider" name="GasHighSlider">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>30</y>
        <width>20</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>1023</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSlider" name="GasLowSlider">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>30</y>
        <width>20</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="GasLowSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="SetValuesButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Values</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReadValuesButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Values</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="ReadingsGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Readings</string>
     </property>
     <widget class="QSpinBox" name="GasHighSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="GasLowSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="BrakeHighSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="BrakeLowSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>800000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ClutchHighSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ClutchLowSpinBox_Read">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClutchLowSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ClutchLowSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClutchLowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ClutchLowSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClutchHighSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ClutchHighSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClutchHighSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ClutchHighSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrakeLowSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrakeLowSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrakeLowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrakeLowSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrakeHighSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrakeHighSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrakeHighSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrakeHighSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GasLowSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GasLowSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GasLowSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GasLowSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GasHighSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GasHighSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GasHighSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GasHighSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
